### PR TITLE
[FR] Localize `Policy as Code`

### LINKED ISSUE #2563: [FR] Localize `Policy as Code`
Localize `Policy as Code` in French

### PR DESCRIPTION
**Pull request by sestegra**
### Describe your changes

Localize `Policy as Code`

### Related issue number or link (ex: `resolves #issue-number`)

Resolved #2563 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/policy-as-code.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: Politique en tant que code
+status: Completed
+category: concept
+tags: ["méthodologie", "", ""]
+draft: 
+---
+
+## Ce que c'est
+
+La politique en tant que code est la pratique de stocker la définition des politiques sous forme d'un ou plusieurs fichiers dans un format lisible et exploitable par une machine.
```
Comment (guillaumebernard84) [inline content/fr/policy-as-code.md:11]: J'aurais plutôt dit: lisible et exploitable par un programme informatique.

File: content/fr/policy-as-code.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Cela remplace le modèle traditionnel où les politiques sont documentées sous une forme lisible par l'homme dans des documents distincts.
+
+## Problème auquel il répond
+
+La construction d'applications et d'infrastructures est souvent contrainte par de nombreuses politiques définies par une organisation,
+par exemple, des politiques de sécurité qui interdisent de stocker des secrets dans le code source, d'exécuter un conteneur avec des permissions de super-utilisateur
```
Comment (guillaumebernard84) [inline content/fr/policy-as-code.md:17]: C’est du pinaillage, mais il me semble que l'on peut enlever la virgule derrière "par example", et que l'on peut en ajouter une derrière  "super-utilisateur".

File: content/fr/policy-as-code.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ou de stocker des données en dehors d'une région géographique spécifique.
+Il est très laborieux et sujet aux erreurs pour les développeurs et les relecteurs de vérifier manuellement les applications et les infrastructures par rapport aux politiques documentées.
+Les processus manuels ne peuvent pas répondre aux exigences de réactivité et de mise à l'échelle des applications Cloud Natives.
+
+## Quel en est l'utilité
+
+La description des politiques sous forme de code permet la répétabilité et réduit les erreurs (contrairement à ce qui se fait manuellement).
+Un autre avantage de la politique en tant que code est que le code peut être géré par un système de contrôle de version comme Git. 
+Git crée un historique des journaux de modifications, ce qui est particulièrement utile lorsque quelque chose ne fonctionne pas comme prévu.
```
Comment (guillaumebernard84) [inline content/fr/policy-as-code.md:26]: J'aurais mis "un historique des modifications" au lieu de "un historique des journaux de modifications"

**Review by guillaumebernard84 — CHANGES_REQUESTED** [review]

File: content/fr/policy-as-code.md (hunk 1 above)
Comment (sestegra) [reply content/fr/policy-as-code.md:11]: Oui, je préfère aussi.

File: content/fr/policy-as-code.md (hunk 2 above)
Comment (sestegra) [reply content/fr/policy-as-code.md:17]: C'est correct, la virgule avant suffit.

File: content/fr/policy-as-code.md (hunk 3 above)
Comment (sestegra) [reply content/fr/policy-as-code.md:26]: C'est plus clair effactivement

**Review by guillaumebernard84 — APPROVED** [review]
LGTM 🚀

**Review by Krast76 — APPROVED** [review]
LGTM 🚀

**Pull request merged**